### DISCUSSION TITLE
skoala/trad-ms/hosted/services/auth

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3547] [Announcements]
# skoala/trad-ms/hosted/services/auth

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/trad-ms/hosted/services/auth.html

**Comment by zhoujw1974** [comment]
授权规则开业根据资源的请求来源限制资源是否通过， 是不是应该为：“ 授权规则可以根据资源的请求来源限制资源是否通过。”？
